src/geo/map.js

```diff
@@ -504,35 +504,37 @@ var Map = Model.extend({
   },
 
   getEstimatedFeatureCount: function () {
-    if (this.hasEstimatedFeatureCount()) {
-      return _.reduce(this.layers.getCartoDBLayers(), function (memo, layerModel) {
-        return memo + layerModel.getEstimatedFeatureCount();
-      }, 0);
+    var acum = 0;
```

Comment:
Fine by me, but the name of the function could be changed because the purpose has changed 😬 .